6 HW task ready

Scofa26 opened the pull request.

Scofa26 committed:
6 HW task ready

Scofa26 committed:
edit TargetFramework

wildpioneer commented on AQA_MTS/Program.cs:
```diff
@@ -6,131 +6,216 @@ class Program
-            Console.WriteLine($"Last item in array: {array[array.GetUpperBound(0)]}");
+             for (int i = 0; i < array.Length; i++)
+             {
+                 if (array[i] == number) flag = true;
```
А если мы уже нашли вхождение элемента - то зачем нам дальше продолжать поиск?

Scofa26 replied:
![image](https://github.com/Scofa26/AQA_MTS/assets/125700209/1c5f2a98-dea3-4088-80e5-70af29d63664)
Исправила - добавила break

wildpioneer commented on AQA_MTS/Program.cs:
```diff
@@ -6,131 +6,216 @@ class Program
+             {
+                 for (int i = 0; i < n; i++)
+                 {
+                     if (i == j) continue;
```
Вот тут совсем не понял - нужно пояснение с твоей стороны

Scofa26 replied:
В верхнем цикле я искала индекс подходящего по условие задачи числа, т.е. которое нужно удалить. И далее я решила создать копию исходного массива, но без указанного числа. И тут как раз я проверяю, если индекс текущего равен индексу числа, которого нужно удалить, я континью, тем самым пропуская это число к добавлению

wildpioneer commented on AQA_MTS/Program.cs:
```diff
@@ -6,131 +6,216 @@ class Program
+
+                      }
+
+                      Console.WriteLine($" \n max = {max} \n min = {min} \n Sum/n = {sum / n}");
```
Средний элемент это не среднее арифметическое
1 5 10  
Средний 5
Среднее арифметическое 5.3

Scofa26 replied:
В этой задаче я решила найти и среднее и ср. арифметическое 
как раз ниже в строках 137 - 140 я нахожу и вывожу среднее 
![image](https://github.com/Scofa26/AQA_MTS/assets/125700209/337d585a-9eff-4333-8d6d-55bd1e0f37fc)

wildpioneer commented on AQA_MTS/Program.cs:
```diff
@@ -6,131 +6,216 @@ class Program
+             {
+                 array3[i] = new Random().Next(10);
+                 array4[i] = new Random().Next(10);
+
```
Форматирование - пустые строки - следи за ними внимательно

Scofa26 replied:
удалила пустую строку, но пришлось вручную, потому что стандартное форматирование VS этого не сделало (

wildpioneer commented on AQA_MTS/Program.cs:
```diff
@@ -6,131 +6,216 @@ class Program
+
+             }
+
+             if (Convert.ToDouble(sum3 / 2.0) == Convert.ToDouble(sum4 / 2.0)) Console.WriteLine("Средние арифметические равны ");
```
1. Так - тут опять не понял - почему ты сумму делишь на 2? Ведь среднее арифметическое это сумма / количество элементов
2. Нужно расчет среднего арифметического вынести в переменные и посчитать до проверки

Scofa26 replied:
Да, тут ошиблась и делила на 2. Исправила ) 
![image](https://github.com/Scofa26/AQA_MTS/assets/125700209/c77c9999-61ff-44f9-afba-b6b88907fb2a)

Scofa26 replied:
Да, тут ошиблась , исправила )

Scofa26 committed:
fix HW
добавила break в цикл ( строки 25-27)
исправила нахождение среднего арифметического в строках 179 - 184

wildpioneer commented:
Проверено
